framework/python/src/common/risk_profile.py

```diff
@@ -70,6 +70,8 @@ def update(self, profile_json, profile_format):
     self.questions = new_profile.questions
     self.status = new_profile.status
 
+    self.update_risk(profile_format=profile_format)
+
```

Comment:
Should be 

`self.risk = new_profile.risk `

since this will already be calculated, no need to do it again

Reply:
Changed